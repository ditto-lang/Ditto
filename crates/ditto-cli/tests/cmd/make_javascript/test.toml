bin.name = "ditto"
args = ["make"]
fs.sandbox = true
