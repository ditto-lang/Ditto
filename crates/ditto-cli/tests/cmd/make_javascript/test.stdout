Checking JsPkg
Generating JavaScript for JsPkg
